### DISCUSSION TITLE
Running a relay?

### DISCUSSION THREAD

**Discussion by Kruwed** [guardianproject/orbot-android#1171] [General]
I notice in my settings there's a "Enable your device to be a non-exit relay". As a heavy user of Tor, I would be happy to contribute resources back to the network. However, I'm looking for evidence that there's any benefit to enabling this setting. Will my mobile phone stay 'awake' long enough to relay effectively, or will it be too unstable for it to be useful?

The default nickname is "OrbotRelay" but a quick search doesn't seem to show any active: https://metrics.torproject.org/rs.html#search/OrbotRelay

**Comment by bitmold** [comment]
We disabled this feature, An Android device shifting networks throughout the day is a bad device to use as a relay.
